dags/aps/aps.py

```diff
@@ -27,18 +27,21 @@ def save_json_in_s3(dates: dict, repo: IRepository = APSRepository()):
         rest_api = APSApiClient(
             base_url=os.getenv("APS_API_BASE_URL", "http://harvest.aps.org")
         )
-        articles_metadata = str.encode(
-            json.dumps(rest_api.get_articles_metadata(parameters))
-        )
-        return save_file_in_s3(data=articles_metadata, repo=repo)
+        articles_metadata = rest_api.get_articles_metadata(parameters)
+        if articles_metadata is not None:
+            articles_metadata = str.encode(
+                json.dumps(rest_api.get_articles_metadata(parameters))
+            )
+            return save_file_in_s3(data=articles_metadata, repo=repo)
+        return None
 
     @task()
     def trigger_file_processing(key, repo: IRepository = APSRepository()):
         if key is None:
             logging.warning("No new files were downloaded to s3")
             return
         ids_and_articles = split_json(repo, key)
-        trigger_file_processing_(ids_and_articles)
+        return trigger_file_processing_(ids_and_articles)
```

Comment:
I just called two different functions with the same name, so I just added "_". Will rename it. 
trigger_file_processing_ returns because in the xcom section in UI we can see which articles were passed to the following DAG. Useful for following that is happening